src/VirtoCommerce.ShippingModule.Core/Services/IShippingMethodsService.cs

```diff
@@ -1,13 +1,6 @@
-using System.Threading.Tasks;
-using VirtoCommerce.ShippingModule.Core.Model;
-
 namespace VirtoCommerce.ShippingModule.Core.Services
 {
     public interface IShippingMethodsService
```

Comment:
Ensure these removed methods are signature compatible with existing ones from ICrudService. Otherwise, compatibility will be broken.